### PR TITLE
Team spotlight blog post - II

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: blog/news-and-updates/team-spotlight-ii.mdx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Team spotlight - Internet Identity
+description: Welcome to this installment of team spotlight. This week, the have a very exciting team to dive into - the Internet Identity team!
+tags: [Team spotlight]
+image: /img/blog/dev-update-blog-internet-id.jpg
+---
+
+![Internet Identity Team spotlight](../../static/img/blog/dev-update-blog-internet-id.jpg)
+
+Greetings developers! Welcome to this installment of team spotlight. This week, the have a very exciting team to dive into: the Internet Identity team! As the team behind developing one of the most cutting-edge tools for digital identities, their mission and focus is a personal favorite. Without further adieu, let's get started!
```
Comment (dfx-json) [inline blog/news-and-updates/team-spotlight-ii.mdx:10]:
```suggestion
Greetings developers! Welcome to this installment of team spotlight. This week, we have a very exciting team to dive into: the Internet Identity team! As the team behind developing one of the most cutting-edge tools for digital identities, their mission and focus is a personal favorite. Without further adieu, let's get started! 
```

File: blog/news-and-updates/team-spotlight-ii.mdx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+tags: [Team spotlight]
+image: /img/blog/dev-update-blog-internet-id.jpg
+---
+
+![Internet Identity Team spotlight](../../static/img/blog/dev-update-blog-internet-id.jpg)
+
+Greetings developers! Welcome to this installment of team spotlight. This week, the have a very exciting team to dive into: the Internet Identity team! As the team behind developing one of the most cutting-edge tools for digital identities, their mission and focus is a personal favorite. Without further adieu, let's get started! 
+
+
+**Hello and welcome to this week's issue of DFINITY's team spotlight! This week, we're going to learn more about the team behind Internet Identity and what's in the works for digital identity on the Internet Computer.**
```
Comment (dfx-json) [inline blog/news-and-updates/team-spotlight-ii.mdx:13]: Is it intentional to reiterate the paragraph above here?

File: blog/news-and-updates/team-spotlight-ii.mdx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+**Those collaborations sound very exciting, I'm eager to see the result of those! How does Internet Identity compare to cryptowallets and other forms of digital identity? What makes it unique?**
+
+*Internet Identity offers a convenient and secure way to sign into dapps, while being fully on-chain. It is not a classical crypto wallet, as it does not offer a transaction signing flow (yet). It is more comparable to a classical Web 2.0 login, such as sing-in with Google, Facebook, etc. The main strengths of II are its ease of use and high security, while not burdening the user with key management and other Web 3.0 complexities.*
+
+**I think that is a great way to describe the difference between II and other forms of digital identities. What’s the biggest challenge the team has faced with developing Internet Identity?**
+
+*We are a small team, which enables us to release changes and respond to feedback frequently, but also comes with limitations. It’s been really rewarding to be able to deliver so much with such a small team.*
```
Comment (dfx-json) [inline blog/news-and-updates/team-spotlight-ii.mdx:33]:
```suggestion
*We are a small team, which enables us to release changes and respond to feedback frequently, but [that] also comes with limitations. It’s been really rewarding to be able to deliver so much with such a small team.*
```

File: blog/news-and-updates/team-spotlight-ii.mdx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+**How has the team handled or responded to developer onboarding friction or difficulties using Internet Identity?**
+
+*We have created a [special dev build](https://github.com/dfinity/internet-identity#flavors) of Internet Identity that makes working with II in a local dev environment very easy. In addition, we have created a [step-by-step example](https://github.com/dfinity/examples/tree/master/motoko/internet_identity_integration) showcasing how to use the dev build. In the future we also plan to support the `dfx pull` command.*
```
Comment (dfx-json) [inline blog/news-and-updates/team-spotlight-ii.mdx:37]:
```suggestion
*We have created a [special dev build](https://github.com/dfinity/internet-identity#flavors) of Internet Identity that makes working with II in a local dev environment very easy. In addition, we have created a [step-by-step example](https://github.com/dfinity/examples/tree/master/motoko/internet_identity_integration) showcasing how to use the dev build. In the future we also plan to support the `dfx deps` command.*
```

File: blog/news-and-updates/team-spotlight-ii.mdx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+**Those are some incredible improvements, and I'm sure our dev community has benefited greatly from them! What is the primary project that the Internet Identity team is working on or focused on currently?**
+
+*Right now we are focused on the following projects:*
+
+- *Verifiable Credentials aka Attribute Sharing, this means that through the use of cryptographic protocols, different applications running on IC will be able to share user attributes between each other in a manner that is both secure and privacy-preserving.*
+
+- *Non-passkey authentication, this means that for users where the use of passkeys is not an option (e.g., iPhone users without an iCloud account), we are offering an alternative authentication mechanism.*
+
+- *Generative-based II logos for each Principal, we have redesigned the II logo and each Internet Identity Principal will now have its own individualized and generative-based logo!*
```
Comment (dfx-json) [inline blog/news-and-updates/team-spotlight-ii.mdx:55]:
```suggestion
- *Generative-based II logos for each Principal – we have redesigned the II logo and each Internet Identity Principal will now have its own individualized and generative-based logo!*
```

**Review by dfx-json — APPROVED** [review]

File: blog/news-and-updates/team-spotlight-ii.mdx (hunk 2 above)
Comment (jessiemongeon1) [reply blog/news-and-updates/team-spotlight-ii.mdx]: It was, as I tried to provide both a 'written' into and a 'speaking' intro to signify the beginning of the 'interview', but it does read a bit redundant. I've removed the second paragraph to fix this.

**Pull request merged**